models.py

```diff
@@ -0,0 +1,153 @@
+from datetime import datetime
+from werkzeug.security import generate_password_hash, check_password_hash
+from app import db, login
+from flask_login import LoginManager, UserMixin, login_required
+
+
+# klasa z kontem użytkownika
+class User(UserMixin, db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(64), index=True, unique=True)
```

Comment:
I may be wrong but is `db.String(64)` setting the max length of the username to `64`? If yes, then I didn't see you checking the username in the (registration and login) form and if it is longer than this value thane raising error (informing user). I may be wrong.. maybe I just didn't notice.